### ISSUE TITLE
No paths for "Manual" type projects are listed on 22.0.0

### ISSUE THREAD

**Issue by kurokobo** [ansible/tower-operator#1323]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx-operator/issues) for duplicates.
- [X] I understand that the AWX Operator is open source software provided for free and that I might not receive a timely response.

### Bug Summary

The `awx-web` container in `2.0.0` does not mount the project directory, so when creating a project of type "Manual", no path is shown.

### AWX Operator version

2.0.0

### AWX version

22.0.0

### Kubernetes platform

kubernetes

### Kubernetes/Platform version

K3s v1.26.3+k3s1

### Modifications

no

### Steps to reproduce

1. Create PV and PVC to use as project directory
2. Deploy AWX with `projects_persistence: true` and `projects_existing_claim: <PVC>`

### Expected results

PVC is mounted on `/var/lib/awx/projects` on `awx-web` container and directories that manually created on the PVC can be used for "Manual" type projects

### Actual results

- No PVC is mounted on `/var/lib/awx/projects` on `awx-web` container
- There is no directory `/var/lib/awx/projects` in `awx-web` container
- That PVC is only mounted on `awx-task` container
- Always no paths are listed for "Manual" type project on "Create New Project" page even if some usable directories exist in the PVC

### Additional information

@thedoubl3j @TheRealHaoLiu @relrod 
This is related to https://github.com/ansible/awx-operator/issues/1266 and #1283 

As far as I know in the current implementation the selectable paths on "Create New Project" page for "Manual" type project are _paths under `/var/lib/awx/projects` on `awx-web` container_ itself.
By splitting web/task container in `2.0.0`, PVC for project no longer mounted on `awx-web` container and this change causes this issue.

What do you think, should we let the `awx-web` mount the PVC like the past implementation?

I think it would be smart to implement that `awx-web` receives the list of selectable paths from `awx-task`, since this PVC should be unnecessary for the `awx-web` except when creating Manual type project and Job Template for the project, but I have no idea this can be implemented or not.

As a workaround, mounting same PVC on `awx-web` using `extra_volumes` solves this issue.

```yaml
  extra_volumes: |
    - name: awx-projects
      persistentVolumeClaim:
        claimName: awx-projects-claim

  web_extra_volume_mounts: |
    - mountPath: /var/lib/awx/projects
      name: awx-projects
```

### Operator Logs

N/A

**Comment by thedoubl3j** [comment]
Great catch @kurokobo

**Comment by vfilenga** [comment]
hey @kurokobo thnx for the contribution.

where exactly (file) do you put these yml extra volumes mapping, if you don't mind asking?

**Comment by kurokobo** [comment]
@vfilenga 
This issue occurred only in AWX Operator 2.0.0 and has been fixed in 2.0.1 and later.
If you are on 2.0.1 or later and are still having this issue, it is probably a completely different issue caused by your configuration (or environment) and my workaround will not help.

For usage question, ask chat, mailing list, or folum:

- https://github.com/ansible/awx-operator#get-involved
- https://forum.ansible.com/c/help/6/none

If you are indeed using 2.0.0, place `extra_volumes` and `web_extra_volume_mounts` under `spec` of your `AWX` like this:

```yaml
---
apiVersion: awx.ansible.com/v1beta1
kind: AWX
metadata:
  name: awx-demo
spec:
  service_type: nodeport

  ...(omit)...

  projects_persistence: true
  extra_volumes: |
    - name: awx-projects
      persistentVolumeClaim:
        claimName: awx-projects-claim
  web_extra_volume_mounts: |
    - mountPath: /var/lib/awx/projects
      name: awx-projects
```

**Comment by vfilenga** [comment]
Thx for repplying @kurokobo .

I guess that if we follow the official guide (https://github.com/ansible/awx-operator/blob/devel/docs/installation/basic-install.md) that would be the awx-demo.yml.

![image](https://github.com/ansible/awx-operator/assets/110911100/52bca5a0-9fb6-46bd-bc2c-6ec76edc6424)

I created these directories by hand in the task and web containers and copied the files using kubectl cp.

It seems to be working, but I will try that to see if there's any noticeble difference.
